openTEPES/9n/oT_Data_RESEnergy_9n.csv

```diff
@@ -1,2 +1,2 @@
-﻿,,RESEnergy
+,,RESEnergy
```

Comment:
There appears there's a Byte Order Mark character at the beginning of the file. 

<img width="690" alt="image" src="https://github.com/user-attachments/assets/76f16597-75cf-44db-a4d1-d1044ec26506">

Related: https://stackoverflow.com/a/60517948/5451769

I recommend removing these characters as they can cause issues with parsing.